src/adr/references/documenting-architecture-decisions.md

```diff
@@ -0,0 +1,160 @@
+# [Documenting Architecture Decisions](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions)
+
+## Context
+
+Architecture for agile projects has to be described and defined
```

Comment:
Maybe we can say that its also true for community projects...

Reply:
that's a copy and paste of the original article

Reply:
Dont we need to adapt it?

Reply:
if we want to keep it as a reference, no.